pkg/odo/cli/delete/component.go

```diff
@@ -141,11 +159,19 @@ func (o *ComponentOptions) deleteDevfileComponent() error {
 		}
 
 		// delete all the resources
-		failed := o.clientset.DeleteClient.DeleteResources(resources)
+		failed := o.clientset.DeleteClient.DeleteResources(devfileResources)
 		for _, fail := range failed {
 			log.Warningf("Failed to delete the %q resource: %s\n", fail.GetKind(), fail.GetName())
 		}
 		log.Infof("The component %q is successfully deleted from namespace %q", componentName, namespace)
+
+		if len(remainingResources) != 0 {
+			log.Printf("There are still resources left in the cluster that might be belonging to the deleted component.")
+			for _, resource := range remainingResources {
+				fmt.Printf("\t- %s: %s\n", resource.GetKind(), resource.GetName())
+			}
```

Comment:
Can we use the tabwriter here instead? It takes care of formatting. We have used it in the odo code in the past.